common/src/main/java/com/microsoft/hydralab/performance/inspectors/WindowsBatteryInspector.java

```diff
@@ -23,20 +26,36 @@
  * the elevation process to block the testing. There is a workaround to disable the UAC dialog by setting "Never notify"
  * in the UAC settings panel.
  *
- * TODO:
- * Need to verify if the agent configured with elevated privileges can bypass the UAC popup without changing the UAC
- * configuration.
- * Add a new method in ShellUtils to run the admin command if the TODO is not feasible.
+ * Prerequisites:
+ * 1. Requires a Windows device with a battery, such as a laptop
+ * 2. Need to disable UAC popup manually since the elevated privileges is required which would blocking the performance
+ * testing.
```

Comment:
Do we need to add these requirements to the Wiki?

Reply:
Thanks for comments. Added to https://github.com/microsoft/HydraLab/wiki/Deploy-a-test-agent-service
![image](https://user-images.githubusercontent.com/10386842/223955299-dac341c9-e242-49a5-8449-e8be030c94bc.png)